SwrveSDKCommon/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion '23.0.2'
```

Comment:
Think you had a previous PR for having values pulled in from gradle.properties. Would be good to roll it into this also. :+1: